neuro_flow/batch_runner.py

```diff
@@ -282,7 +276,7 @@ async def inspect(
 
         self._console.print(f"[b]Attempt #{attempt.number}[/b]", attempt.result)
 
-        started, finished = await self._storage.fetch_attempt(attempt)
+        started, running, finished = await self._storage.fetch_attempt(attempt)
```

Comment:
`running` can be calculated from the iteration over `started`. You need to check the actual task status anyway.

Reply:
Done